CRM/Contact/Form/Search/Criteria.php

```diff
@@ -511,9 +538,6 @@ public static function relationship(&$form) {
         ['id' => 'relation_target_group', 'multiple' => 'multiple', 'class' => 'crm-select2']
       );
     }
-    CRM_Core_Form_Date::buildDateRange($form, 'relation_start_date', 1, '_low', '_high', ts('From:'), FALSE, FALSE);
-    CRM_Core_Form_Date::buildDateRange($form, 'relation_end_date', 1, '_low', '_high', ts('From:'), FALSE, FALSE);
-
     CRM_Core_Form_Date::buildDateRange($form, 'relation_active_period_date', 1, '_low', '_high', ts('From:'), FALSE, FALSE);
```

Comment:
@eileenmcnaughton not wanting to convert this one yet?

Reply:
@seamuslee001 nope - next round